src/build.jl

```diff
@@ -215,32 +215,62 @@ function solve!(m::AbstractMPSGEModel; kwargs...)
 
 
     for (k,v) in kwargs
-
         JuMP.set_attribute(jm, string(k), v)
     end
 
+    # Unfix numeraire, if set
+    if !ismissing(m.numeraire) 
+        unfix(numeraire(m))
+    end
+
+    # Update consumer start values
+    for consumer∈consumers(m)
+        if termination_status(jm) == OPTIMIZE_NOT_CALLED
+            starting_values = Dict(get_variable(var) => start_value(var) for  var in all_variables(jm))
+            new_start = sum(
+                raw_quantity(
+                    i->get(starting_values, i,missing), 
+                    endowment
+                ) 
+                for (_,endowment)∈endowments(demand(consumer))
+            )
+            new_start += -value(
+                i->get(starting_values,i,missing), 
+                sum(tau(sector, consumer) for sector in production_sectors(m))
+            )
+        else
+            new_start = sum(raw_quantity(e) for (_,e)∈endowments(demand(consumer)))
+            new_start += -value(
+                sum(tau(sector, consumer) for sector in production_sectors(m))
+            )
+        end
+        set_start_value(consumer, new_start)
```

Comment:
This is the starting value for each consumer. Sum of endowments minus taxes.

Reply:
Hmm. I don't think that's right if I understand what's going on here. I think that a consumer can start with some tax transfers at the benchmark. e.g. that's true for the WiNDC national model.